sdk/core/azure-core-tracing-opencensus/src/samples/AsyncQueueClientEnqueueMessages.md

```diff
@@ -14,7 +14,7 @@ Sample uses **[opencensus-impl][opencensus_impl]** as implementation package and
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-storage-queue</artifactId>
-  <version>12.1.0</version>
+  <version>12.2.0-beta.2</version>
```

Comment:
I think this should be marked as a dependency version and we should update how these samples are injected into the README to ensure they compile and use the right version.